src/wasm-lib/kcl/src/parser/snapshots/kcl_lib__parser__parser_impl__snapshot_tests__af.snap

```diff
@@ -277,6 +277,7 @@ expression: actual
           "digest": null
         }
       ],
+      "visibility": "default",
```

Comment:
If you set `visibility`'s default to "default" and then use serde annotations to skip serializing if the value is "default", then all the test snapshots will stay the same and the PR will be smaller (and the snapshots will be easier to read)

Reply:
Done. 3684acc2201afc9d1c46c412ddef126574bd5183 567e21fecf822596531bcae6fa6e202c21aeb489